container/libcontainer/helpers_test.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var defaultCgroupSubsystems = []string{
-	"systemd", "freezer", "memory", "blkio", "hugetlb", "net_cls,net_prio", "pids", "cpu,cpuacct", "devices", "cpuset", "perf_events",
+	"systemd", "freezer", "memory", "misc", "blkio", "hugetlb", "net_cls,net_prio", "pids", "cpu,cpuacct", "devices", "cpuset", "perf_events",
```

Comment:
"misc" cgroup was added in Linux 5.13.  Is it something available already on all LTS distros?

(If not, might be better to drop it from defaults.)

Reply:
I dropped this.